Views/page_views.view.lkml

```diff
@@ -20,11 +21,11 @@ view: page_views {
 
   # Addititonal Drills
   dimension: browser_language {
-    drill_fields: [page_display_url]
+    drill_fields: [shared_fields_no_session.page_display_url]
```

Comment:
@BJenning drills on `page_display_url` do not work when scoped like this. I think all you need to do is remove the "`shared_fields_no_session.`" scope prefixing `page_display_url`. You may need to do so on all the dimensions using `page_display_url` as a drill fields.

Please test this in an explore like https://analytics.gov.bc.ca/explore/snowplow_web_block/page_views?qid=uCCZkdMnkEIV2rJRV2goqu&toggle=fil before committing.

Reply:
Ive removed the filename. Its now appearing in drill down options.